frontend/packages/topology/src/components/graph-view/TopologyShortcuts.tsx

```diff
@@ -7,39 +7,43 @@ export type Options = {
   supportedFileTypes: string[];
   isEmptyModel: boolean;
   viewType: TopologyViewType;
+  allImportAccess: boolean;
 };
-export const getTopologyShortcuts = (t: TFunction, options: Options): React.ReactElement => (
-  <ShortcutTable>
-    {!options.isEmptyModel && options.viewType === TopologyViewType.graph && (
-      <>
-        <Shortcut data-test-id="move" drag>
-          {t('topology~Move')}
+export const getTopologyShortcuts = (t: TFunction, options: Options): React.ReactElement => {
+  const { supportedFileTypes, isEmptyModel, viewType, allImportAccess } = options;
+  return (
+    <ShortcutTable>
+      {!isEmptyModel && viewType === TopologyViewType.graph && allImportAccess && (
+        <>
+          <Shortcut data-test-id="move" drag>
+            {t('topology~Move')}
+          </Shortcut>
```

Comment:
I think we should still show `Move` or did you notice a bug when the user moves things in topology? The result are stored in the user settings which should still works.

Reply:
+1